table/namespace_classifier_test.go

```diff
@@ -154,14 +154,16 @@ func (s *testTableNamespaceSuite) TestNamespaceOperation(c *C) {
 
 	// Add tableID
 	err = tableClassifier.AddNamespaceTableID("test1", 1)
-	namespaces = tableClassifier.GetNamespaces()
 	c.Assert(err, IsNil)
+	namespaces = tableClassifier.GetNamespaces()
+	c.Assert(len(namespaces), Equals, 1)
```

Comment:
can we use `c.Assert(namespaces, HasLen, 1)` here?

Reply:
OK.